docs/release/release-process.md

```diff
@@ -192,6 +201,77 @@ success, all release artifacts will be officially published.
 After the release build succeeds, verify that the correctly versioned Stack images have been pushed
 to Docker Hub.
 
+### Template Stack Release Process
+
+The Template Stacks we maintain are slightly different from the
+controller-based stacks that we maintain. Their processes are similar
+but a little simpler. This section will walk through how to release the
+Template Stacks themselves, and does not directly apply to core
+Crossplane.
+
+For Template Stacks, we do not use release branches unless we need to
+make a patch release. In the future we may need a more robust branching
+strategy, but for now we are not using branches because it is simpler.
+
+Note that Template Stacks **do not** require any code changes to update
+the version.
+
+### Template Stack Tag And Publish Pipeline
+
+Here is the list of all template stacks:
+
+* `stack-minimal-gcp`
```

Comment:
can you link each of these to their repo, similar to how we are doing it in https://github.com/crossplane/crossplane/blob/master/docs/release/release-process.md#scope?

Reply:
This is done